src/Symfony/Bundle/FrameworkBundle/Resources/config/serializer.xml

```diff
@@ -1,181 +0,0 @@
-        </service>
-
-        <service id="serializer.normalizer.datetime" class="Symfony\Component\Serializer\Normalizer\DateTimeNormalizer">
-            <!-- Run before serializer.normalizer.object -->
-            <tag name="serializer.normalizer" priority="-910" />
-        </service>
-
-        <service id="serializer.normalizer.json_serializable" class="Symfony\Component\Serializer\Normalizer\JsonSerializableNormalizer">
-            <!-- Run before serializer.normalizer.object -->
-            <tag name="serializer.normalizer" priority="-900" />
-        </service>
-
-        <service id="serializer.normalizer.problem" class="Symfony\Component\Serializer\Normalizer\ProblemNormalizer">
-            <argument>%kernel.debug%</argument>
-            <!-- Run before serializer.normalizer.object -->
-            <tag name="serializer.normalizer" priority="-890" />
-        </service>
-
-        <service id="serializer.denormalizer.unwrapping" class="Symfony\Component\Serializer\Normalizer\UnwrappingDenormalizer">
-            <argument type="service" id="serializer.property_accessor" />
-            <!-- Run before serializer.normalizer.object -->
-            <tag name="serializer.normalizer" priority="1000" />
-        </service>
-
-        <service id="serializer.normalizer.object" class="Symfony\Component\Serializer\Normalizer\ObjectNormalizer">
-            <argument type="service" id="serializer.mapping.class_metadata_factory" />
-            <argument type="service" id="serializer.name_converter.metadata_aware" />
-            <argument type="service" id="serializer.property_accessor" />
-            <argument type="service" id="property_info" on-invalid="ignore" />
-            <argument type="service" id="serializer.mapping.class_discriminator_resolver" on-invalid="ignore" />
-            <argument>null</argument>
-            <argument type="collection" />
-
-            <!-- Run after all custom normalizers -->
-            <tag name="serializer.normalizer" priority="-1000" />
-        </service>
-        <service id="Symfony\Component\Serializer\Normalizer\ObjectNormalizer" alias="serializer.normalizer.object" />
-
-        <service id="serializer.denormalizer.array" class="Symfony\Component\Serializer\Normalizer\ArrayDenormalizer">
-            <!-- Run before the object normalizer -->
-            <tag name="serializer.normalizer" priority="-990" />
-        </service>
-
-        <!-- Loader -->
-        <service id="serializer.mapping.chain_loader" class="Symfony\Component\Serializer\Mapping\Loader\LoaderChain">
-            <argument type="collection" />
-        </service>
-
-        <!-- Class Metadata Factory -->
-        <service id="serializer.mapping.class_metadata_factory" class="Symfony\Component\Serializer\Mapping\Factory\ClassMetadataFactory">
-            <argument type="service" id="serializer.mapping.chain_loader" />
-            <argument>null</argument>
```

Comment:
this arg does not exist. I'll create a PR to fix this. Please rebase this branch afterwards.

Reply:
#37214